core/commands/add.go

```diff
@@ -44,6 +45,7 @@ remains to be implemented.
 	Options: []cmds.Option{
 		cmds.OptionRecursivePath, // a builtin option that allows recursive paths (-r, --recursive)
 		cmds.BoolOption(quietOptionName, "q", "Write minimal output"),
+		cmds.BoolOption(silentOptionName, "x", "Write no output"),
```

Comment:
i like this. should we do something like `-qq` (kind of like the typical unixy `-v, -vv, -vvv` crap.). would be nice to have the same behavior everywhere:

```
[none]    normal output
-q        minimal ouput
-qq       no output
```
- **argument against `-qq`** - it sounds kind of dumb. (have always thought `-vvvv` "verbose * 4" is very silly)
- **argument against `-{x, s, }`** - would be nice to have the same convention everywhere without burning another character in every single command. `-x` and `-s` are likely to make sense in various places.
- **argument against shorthand** - maybe we dont use `-qq` or `-{s, x}`, but force a full `--silent` for no output.